pkg/handlers/apk_keywords.go

```diff
@@ -0,0 +1,1672 @@
+		infura.Scanner{},
+		alchemy.Scanner{},
+		blocknative.Scanner{},
+		moralis.Scanner{},
+		bscscan.Scanner{},
+		percy.Scanner{},
+		tineswebhook.Scanner{},
+		pulumi.Scanner{},
+		databrickstoken.Scanner{},
+		supabasetoken.Scanner{},
+		nugetapikey.Scanner{},
+		aiven.Scanner{},
+		prefect.Scanner{},
+		buildkitev2.Scanner{},
+		opsgenie.Scanner{},
+		dockerhubv1.Scanner{},
+		couchbase.Scanner{},
+		envoyapikey.Scanner{},
+		github_oauth2.Scanner{},
+		snowflake.Scanner{},
+		huggingface.Scanner{},
+		trufflehogenterprise.Scanner{},
+		salesforce.Scanner{},
+		sourcegraph.Scanner{},
+		tailscale.Scanner{},
+		loggly.Scanner{},
+		web3storage.Scanner{},
+		&ramp.Scanner{},
+		&anthropic.Scanner{},
+		&sourcegraphcody.Scanner{},
+		voiceflow.Scanner{},
+		ip2location.Scanner{},
+		grafanaserviceaccount.Scanner{},
+		vagrantcloudpersonaltoken.Scanner{},
+		openvpn.Scanner{},
+		&metabase.Scanner{},
+		appoptics.Scanner{},
+		zerotier.Scanner{},
+		betterstack.Scanner{},
+		coinbase_waas.Scanner{},
+		replyio.Scanner{},
+		stripo.Scanner{},
+		lemonsqueezy.Scanner{},
+		denodeploy.Scanner{},
+		budibase.Scanner{},
+		requestfinance.Scanner{},
+		coda.Scanner{},
+		grafana.Scanner{},
+		logzio.Scanner{},
+		eventbrite.Scanner{},
+		&overloop.Scanner{},
+		ngrok.Scanner{},
+		replicate.Scanner{},
+		privacy.Scanner{},
+		instamojo.Scanner{},
+		klaviyo.Scanner{},
+		portainer.Scanner{},
+		rabbitmq.Scanner{},
+		planetscale.Scanner{},
+		portainertoken.Scanner{},
+		pagarme.Scanner{},
+		planetscaledb.Scanner{},
+		azure.Scanner{},
+		azurestorage.Scanner{},
+		azurecontainerregistry.Scanner{},
+		azurebatch.Scanner{},
+		// azurefunctionkey.Scanner{}, // detector is throwing some FPs
+		azuredevopspersonalaccesstoken.Scanner{},
+		azuresearchadminkey.Scanner{},
+		azuresearchquerykey.Scanner{},
+		googleoauth2.Scanner{},
+		dockerhubv2.Scanner{},
+		&jupiterone.Scanner{},
+		gcpapplicationdefaultcredentials.Scanner{},
+		wiz.Scanner{},
+		onfleet.Scanner{},
+		intra42.Scanner{},
+		groq.Scanner{},
+		twitterconsumerkey.Scanner{},
+		eraser.Scanner{},
+		larksuite.Scanner{},
+		larksuiteapikey.Scanner{},
+		endorlabs.Scanner{},
+		atlassianv1.Scanner{},
+		atlassianv2.Scanner{},
+		netsuite.Scanner{},
+		box.Scanner{},
+		robinhoodcrypto.Scanner{},
+		nvapi.Scanner{},
+		railwayapp.Scanner{},
+		meraki.Scanner{},
+		saladcloudapikey.Scanner{},
+		boxoauth.Scanner{},
+		apimetrics.Scanner{},
+		captainDataV1.Scanner{},
+		captainDataV2.Scanner{},
+	}
+
+	// Remove keywords that cause lots of false positives
+	exclusions := []string{
+		"AKIA", "SG.", "pat", "token", "gh", "github", "sql", "database", "http", "key", "api-", "sdk-", "float", "-us", "gh", "pat", "token", "sid", "http", "private", "key", "segment", "close", "protocols", "verifier", "box", "privacy", "dm", "sl.", "vf", "flat",
+	}
+
+	// Create a map of keywords
+	keywords := map[string]struct{}{}
+
+	// Get all Keywords() from each detector
+	for _, detector := range detectorList {
```

Comment:
Can you just load from this instead? https://github.com/trufflesecurity/trufflehog/blob/main/pkg/engine/defaults.go#L814

Reply:
Unfortunately no. Circular import issues. IMO it logically makes sense to move the defaults to the detectors package but I didn’t know if there would be unintended side effects. Our Postman source needed the same data but because it had access to an engine object we could work around it. But we can’t do the same here.

Reply:
I think you’ll need to create a new package, like `defaults`, and move `defaults.go` there. If you move it into the `detectors` package, you’ll still run into circular import issues. I've tried it before 😅

Reply:
Is creating a new `defaults` package something I should do in this PR?

Reply:
I think it's fine. I moved ahocorasick from `engine to `engine/ahocorasick` in #2342 to fix a similar issue.